cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -301,7 +331,8 @@ protected TimeSeries getTimeseries(String page, int pageSize, String names, Stri
                             tsMetadata.getValue("interval", Long.class)),
                     verticalDatumInfo,
                     tsMetadata.getValue(AV_CWMS_TS_ID2.INTERVAL_UTC_OFFSET).longValue(),
-                    tsMetadata.getValue(tzName)
+                    tsMetadata.getValue(tzName),
+                    versionDate, dateVersionType
```

Comment:
Should AGGREGATE get replaced with UNVERSIONED here if the requested time series isn't versioned?

Reply:
It could. If the user passes in null for the versionDate I would have to make a preliminary SQL query to check if the timeseries is versioned right?

Reply:
Unable to perform this update without another db call, waiting on approval to do so

Reply:
You can call `cwms_ts.is_tsid_versioned`

Reply:
Done